content/en/docs/project-user-guide/application-workloads/routes.md

```diff
@@ -0,0 +1,156 @@
+
+     ![obtain-address-nodeport](/images/docs/project-user-guide/application-workloads/routes/obtain-address-nodeport.png)
+
+   * If the [Route access method](#configure-the-route-access-method) is set to LoadBalancer, the gateway address is assigned by the load balancer plugin.
+
+     ![obtain-address-loadbalancer](/images/docs/project-user-guide/application-workloads/routes/obtain-address-loadbalancer.png)
+
+## Configure Domain Name Resolution
+
+If **Auto Generate** is selected in the [route rule configuration](#step-2-configure-route-rules), you do not need to configure domain name resolution and the domain name is automatically resolved by [nip.io](https://nip.io/) into the gateway address.
+
+If **Specify Domain** is selected in the [route rule configuration](#step-2-configure-route-rules), you need to configure domain name resolution on your DNS server or add `<Route gateway address> <Route domain name>` to the `hosts` file of your client machine.
+
+## Access the Route
+
+### NodePort access method
+
+1. Log in to a client machine connected to the Route gateway address.
+
+2. Use the `<Route domain name>:<NodePort>/<Service path>` address to access the back-end service of the Route.
+
+   ![access-route-nodeport](/images/docs/project-user-guide/application-workloads/routes/access-route-nodeport.png)
+
+### LoadBalancer access method
+
+1. Log in to a client machine connected to the Route gateway address.
+
+2. Use the `<Route domain name>/<Service path>` address to access the back-end service of the Route.
+
+   ![access-route-loadbalancer](/images/docs/project-user-guide/application-workloads/routes/access-route-loadbalancer.png)
+
+{{< notice note >}}
+
+If you need to access the Route from outside your private network by using either NodePort or LoadBalancer, depending on your network environment:
+
+* You may need to configure traffic forwarding and firewall policies in your infrastructure environment so that the gateway address and port number of the Route can be accessed.
+* If **Auto Generate** is selected in the [route rule configuration](#step-2-configure-route-rules), you may need to manually [edit the Route rules](#edit-the-route) to change the gateway address in the Route domain name to the external IP address of your private network.
+* If **Specify Domain** is selected in the [route rule configuration](http://localhost:1313/docs/project-user-guide/application-workloads/routes/#step-2-configure-route-rules), you may need to change the configuration on your DNS server or in the `hosts` file of your client machine so that the domain name can be resolved into the external IP address of your private network. 
+
+{{</ notice >}}
+
+## Edit the Route
+
+1. Choose **Application Workloads** > **Routes** on the left navigation bar and click the name of the Route on the right.
```

Comment:
Choose **Application Workloads** > **Routes** on the left navigation bar
->
Choose **Routes** in **Application Workloads** on the left navigation bar